src/main/java/com/rajat/mywebapp/controllers/CustomerController.java

```diff
@@ -32,7 +32,7 @@ public List<CustomerDTO> getAllCustomers() {
         return service.getAllCustomers();
     }
 
-    @RequestMapping(params = "custId",method = RequestMethod.GET)
+    @GetMapping("customers/id")
```

Comment:
There is a mismatch in the parameter mentioned in the get mapping and in the function argument.